pom.xml

```diff
@@ -319,17 +306,6 @@
       <version>1.0.0</version>
     </dependency>
     <!--
-      This depndency has been added to provided scope as it is needed for executing build
-      specific groovy scripts using gmaven+ and not required for downstream project building
-      with spark.
-    -->
-    <dependency>
```

Comment:
Note that removing this from the dependencies list will remove Groovy as a provided dependency from all of the Spark project POMs.  I always thought it was really confusing how the published Maven artifacts would declare a dependency on Groovy, so I'm glad to see this cleaned up.